### PR TITLE
Build binaries for IBM Z (s390x) architecture

### PR DESCRIPTION
**Pull request by aosadchyy**
Updated .travis.yml file to build container images for IBM Z (s390x) in Travis CI. Also executing unit tests on s390x. Travis CI jobs for both amd64 and s390x passed on the fork. 
image

### CODE REVIEW

**Comment by nathanleviere** [comment]
Looking here first - https://github.com/razee-io/RemoteResourceS3/pull/179 and will get to Z next.

**Comment by aosadchyy** [comment]
> Looking here first - [razee-io/RemoteResourceS3#179](https://github.com/razee-io/RemoteResourceS3/pull/179) and will get to Z next.

Sounds good. Both are similar and additive.

**Comment by alewitt2** [comment]
@aosadchyy please pull in master and replicate what we did with PowerPC for IBM Z. eg. please create another stage in the build after the ppc64le one, and add your image to the manifest in the final stage

**Comment by aosadchyy** [comment]
@alewitt2 I've pulled from the master and merged s390x to it. Added a job for s390x and extended the manifest to publish s390x along with other architectures. Please review and let me know if any comments.

File: .travis.yml [hunk 1]
```diff
@@ -122,4 +161,4 @@ jobs:
          on:
            tags: true
            condition: ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$
-     
+
```
Comment (alewitt2) [inline .travis.yml:164]:
```suggestion


```

**Review by alewitt2 — APPROVED** [review]

**Pull request merged**